Documentation/gettingstarted_kafka.rst

```diff
@@ -0,0 +1,392 @@
+
+
+Step 0: Install kubectl & minikube
+==================================
+
+This guide uses `minikube <https://kubernetes.io/docs/getting-started-guides/minikube/>`_
+to demonstrate deployment and operation of Cilium + Kafka in a single-node Kubernetes cluster.
+The minikube VM requires approximately 2 GB of RAM and supports hypervisors like VirtualBox
+that run on Linux, macOS, and Windows.
+
+If you instead want to understand the details of
+deploying Cilium on a full fledged Kubernetes cluster, then go straight to
+:ref:`admin_install_daemonset`.
+
+Install ``kubectl`` version ``>= 1.6.3`` as described in the `Kubernetes Docs
+<https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_.
+
+Install one of the `hypervisors supported by minikube <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_.
+
+Install ``minikube`` ``>= 0.22.3`` as described on `minikube's github page
+<https://github.com/kubernetes/minikube/releases>`_.
+
+Then, boot a minikube cluster with the Container Network Interface (CNI) network plugin enabled:
+
+::
+
+    $ minikube start --network-plugin=cni
+
+After minikube has finished  setting up your new Kubernetes cluster, you can
+check the status of the cluster by running ``kubectl get cs``:
+
+::
+
+    $ kubectl get cs
+    NAME                 STATUS    MESSAGE              ERROR
+    controller-manager   Healthy   ok
+    scheduler            Healthy   ok
+    etcd-0               Healthy   {"health": "true"}
+
+If you see output similar to this, you are ready to proceed to installing Cilium.
+
+Step 1: Installing Cilium
+=========================
+
+The next step is to install Cilium into your Kubernetes cluster.  Cilium installation
+leverages the `Kubernetes Daemon Set <https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/>`_
+abstraction, which will deploy one Cilium pod per
+cluster node.   This Cilium pod will run in the ``kube-system`` namespace along with
+all other system relevant daemons and services.  The Cilium pod will run both the Cilium
+agent and the Cilium CNI plugin.
+
+To deploy Cilium, run:
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes/cilium.yaml
+    clusterrole "cilium" created
+    serviceaccount "cilium" created
+    clusterrolebinding "cilium" created
+    configmap "cilium-config" created
+    secret "cilium-etcd-secrets" created
+    daemonset "cilium" created
+
+Kubernetes is now deploying Cilium with its RBAC, ConfigMap and Daemon Set as a
+pod on all cluster nodes. This operation is performed in the background.
+
+Run the following command to check the progress of the deployment:
+
+::
+
+    $ kubectl get ds --namespace kube-system
+    NAME            DESIRED   CURRENT   READY     NODE-SELECTOR   AGE
+    cilium          1         1         0         <none>          2m
+
+Wait until the cilium Deployment shows a ``CURRENT`` count of ``1`` like above (a READY value of 0 is OK
```

Comment:
Suggestion: use similar formatting for `READY` and value `0`, like `CURRENT` and `1`.